UnitTests/dotNetInstallerLibUnitTests/ConfigFilesUnitTests.cpp

```diff
@@ -110,6 +110,19 @@ void ConfigFilesUnitTests::testSelectLanguageNot1040()
     CPPUNIT_ASSERT(cf[0]->lcid_filter == L"!1040");
 }
 
+void ConfigFilesUnitTests::testSelectLanguageCancelled()

```

Comment:
Something wrong with tabs/spaces here, it's a nitpick but I'd like to keep things consistent, please.